packages/apps/escrow-dashboard/src/constants/index.ts

```diff
@@ -12,7 +25,31 @@ export enum ChainId {
   AVALANCHE_TESTNET = 43113,
   AVALANCHE = 43114,
 }
-
+const rinkeby = {
+  id: ChainId.RINKEBY,
+  name: 'Ethereum Rinkeby',
+  network: 'rinkeby',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'Avalanche',
```

Comment:
you are using Avalanche for rinkeby, anyway Rinkeby network shoudn't be available in the kvstore, this network is deprecated

Reply:
Ah sorry let me change that